.github/workflows/ci.yml

```diff
@@ -14,13 +14,13 @@ jobs:
       - uses: actions/checkout@v3.0.2
       - uses: ros-tooling/setup-ros@v0.4
         with:
-          required-ros-distributions: humble
+          required-ros-distributions: rolling
```

Comment:
I suspect this should still point to `humble`. Please confirm me if that's the case.

Reply:
The repo depends on the events executor whch is only available in iron